docs/configuring-api-console.md

```diff
@@ -18,37 +18,37 @@ console.narrow = true;
-| `proxy` | Sets the proxy URL for the HTTP requests sent from the console. If set all URLs will be altered before sending the data to a transport library by prefixing the URL with this value | `String`
-| `proxy-encodeUrl` | If required by the `proxy` the URL will be URL encoded. | `Boolean` |
-| `no-try-it` | Disables the "try it" button in the method documentation view. The request editor and the response viewer is still available but you must open it programmatically setting `page` property to ` request` | `Boolean` |  
-| `manual-navigation` | Disables navigation in the drawer and renders the navigation full screen, when requested. This is ideal to use in the narrow layouts together with `narrow` property. You will have to set the `navigationOpened` property to `true` or `false` to open / close the navigation. | `Boolean` |
-| `navigation-opened` | If set and `manual-navigation` attribute is set then it will open / close the full screen navigation. | `Boolean` |
+| `aware` | If passing the RAML data with the [raml-aware] element, this is the value of the `scope` attribute used in the aware. | `String` |
+| `page` | Currently selected top level view of the console: either `docs` or `request`. The latter is the "try it" screen. | `String` |
+| `narrow` | Renders the API console in the "narrow" view. This is a mobile-like view. Navigation is hidden in a drawer, and some views are simplified for narrow screens. This view is presented even if the screen is wide enough to display the full console, which facilitates inserting the element as a sidebar of your web page. The `narrow` property is set automatically on mobile devices. | `Boolean` |
+| `append-headers` | Forces the console to send a specific list of headers, overriding user input if needed. The attribute value passes an HTTP headers string separating lines with the `\n` character. Example: `x-api-token: abc\nx-other-headers: header value`. | `String` |
+| `proxy` | Sets the proxy URL for the HTTP requests sent from the console. To alter all URLs before sending the data to a transport library, set the attribute and prefix the URL with its value. | `String`
+| `proxy-encodeUrl` | If required by the `proxy` the URL is URL-encoded. | `Boolean` |
+| `no-try-it` | Disables the "try it" button in the method documentation view. The request editor and the response viewer is still available, but you must open it programmatically by setting `page` property to ` request`. | `Boolean` |  
+| `manual-navigation` | Disables navigation in the drawer and renders the navigation full screen when requested. Use in the narrow layouts with the `narrow` property. Set the `navigationOpened` property to `true` or `false` to open/close the navigation. | `Boolean` |
+| `navigation-opened` | If this and the `manual-navigation` attribute are set, then the full screen navigation will open/close. | `Boolean` |
```

Comment:
I don't get it. 

> If this and the `manual-navigation` attribute are set,

It should be: `If set` - as when the attribute is set, and `the manual-navigation attribute is set`. There's only one attribute.